client/src/components/CommentApp/components/CommentReply/index.tsx

```diff
@@ -243,19 +243,19 @@ export default class CommentReplyComponent extends React.Component<CommentReplyP
     return (
       <>
         <CommentHeader commentReply={reply} store={store} focused={isFocused} />
```

Comment:
Except for the header, everything below is 1:1 the same rendering as for comments. So this would ideally get React refactorings too.